housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/VacuumToolApplication.java

```diff
@@ -124,6 +133,17 @@ public void run(ApplicationArguments args) {
     }
   }
 
+  private void validate(IMetaStoreClient metastore, List<Table> tables) {
+    ValidationResult validateResult = tablesValidator.validate(metastore, tables);
+    if (!validateResult.isValid()) {
+      for (ValidationFailure validationFailure : validateResult.getValidationFailures()) {
+        LOG.error("Table: " + validationFailure.getQualifiedTableName() + ". " + validationFailure.getMessage());
```

Comment:
How about

`LOG.error("Validation error for table '" + validationFailure.getQualifiedTableName() + "': " + validationFailure.getMessage());`

Reply:
sure